velox/exec/fuzzer/RowNumberFuzzerRunner.cpp

```diff
@@ -18,14 +18,48 @@
 #include <gflags/gflags.h>
 #include <gtest/gtest.h>
 #include <unordered_set>
+#include "velox/common/file/FileSystems.h"
 #include "velox/common/memory/SharedArbitrator.h"
 #include "velox/connectors/hive/HiveConnector.h"
 #include "velox/exec/MemoryReclaimer.h"
```

Comment:
The linter complains that these two headers are unnecessary. Could you try removing them?